### PR TITLE
vtkVisualization

### PR DESCRIPTION
**Pull request by dwu12**
Revise both mri image size and colorbar

### CODE REVIEW

File: neurodegeneration/processing-containers/brainvisualize/files/vtkBrainVisual.py [hunk 1]
```diff
@@ -473,6 +419,11 @@ def setup_vtk_pipeline(roi,allz_num,out):
 			setup_camera(viewport_keys[i],ren)
 		elif(viewport_keys[i] == ORIENTATION.COLORBAR): #bottom row, last renderer for colorbar
 			add_scalarbar(ren)
+			ren.ResetCamera()
+			camera =  ren.GetActiveCamera()
+			camera.OrthogonalizeViewUp()
+			ren.ResetCameraClippingRange()	
+			#render_scalarbar(relabelMap,need2relabel)
```
Comment (ashishsingh18) [inline neurodegeneration/processing-containers/brainvisualize/files/vtkBrainVisual.py]: are these needed here? Does it not work without these lines? If they are needed, they should be moved to the function add_scalarbar()

File: neurodegeneration/processing-containers/brainvisualize/files/vtkBrainVisual.py [hunk 2]
```diff
@@ -534,9 +485,18 @@ def _main( roi, allz_num, pdf_path):
...
+	# roi = '/home/diwu/Desktop/kaapana-data-to-check-brainviz/F7/2.16.840.1.114362.1.12066432.24920037488.604832617.475.5969/relabel/2.16.840.1.114362.1.12066432.24920037488.604832617.475.5969.nii.gz'
+	# input = read_itk_image(roi)
+	# output = write_itk_image(input,'/home/diwu/Desktop/kaapana-data-to-check-brainviz/F7/2.16.840.1.114362.1.12066432.24920037488.604832617.475.5969/relabel/2.16.840.1.114362.1.12066432.24920037488.604832617.475.5969.nrrd')
+	# roi_file = '/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/relabel/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168.nrrd'
+	# with open('/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/roi-quantification/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168_allz_num.pkl','rb') as f:
+	# 	allz_num = pickle.load(f)
+	# pdf_path = './test.pdf'
+	# _main( roi_file, allz_num, pdf_path)
+
+	# F2
```
Comment (ashishsingh18) [inline neurodegeneration/processing-containers/brainvisualize/files/vtkBrainVisual.py]: Please remove all commented code

**Review by ashishsingh18 — CHANGES_REQUESTED** [review]
Thanks @dwu12 for the PR. Please check minor suggestions.

File: neurodegeneration/processing-containers/brainvisualize/files/vtkBrainVisual.py (hunk 1 above)
Comment (dwu12) [reply neurodegeneration/processing-containers/brainvisualize/files/vtkBrainVisual.py]: done

File: neurodegeneration/processing-containers/brainvisualize/files/vtkBrainVisual.py (hunk 2 above)
Comment (dwu12) [reply neurodegeneration/processing-containers/brainvisualize/files/vtkBrainVisual.py]: done

File: neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py [hunk 3]
```diff
@@ -239,6 +250,10 @@ def plotWithRef(dfRef, WMLSref, dfSub, fname, spareAD, spareBA):
 			fig.append_trace( go.Scatter(mode='lines', x=XX_AD, y=XX10_AD, legendgroup='AD', marker=dict(color='MediumBlue', size=10), name='10th percentile for AD', line=dict(dash = 'dash'), showlegend=sl),row,column)
 			fig.append_trace( go.Scatter(mode='markers', x=dfSub.Age.tolist(), y=dfSub[selVar].tolist(), legendgroup='Patient', marker=dict(color='Black', symbol = 'circle-cross-open', size=16,line=dict(color='MediumPurple', width=3)), name='Patient', showlegend=sl),row,column)
 
+		# min_ = min(ADRef.Age.min(), CNRef.Age.min())
+		# max_ = min(ADRef.Age.max(), CNRef.Age.max())
+		# fig.update_xaxes(range=[min_, max_])
+
```
Comment (ashishsingh18) [inline neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py]: Please remove commented code. If it is commented for a specific reason, please add a comment about why it is there.

File: neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py [hunk 4]
```diff
@@ -258,7 +273,7 @@ def plotWithRef(dfRef, WMLSref, dfSub, fname, spareAD, spareBA):
 
 # TODO: 
 def makeFlagTablePKL(dfRef,mydict,MUSErois,path):
-	maphemi = pd.read_csv('/refs/MUSE_ROI_Dictionary.csv')
+	maphemi = pd.read_csv('../refs/MUSE_ROI_Dictionary.csv')
```
Comment (ashishsingh18) [inline neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py]: Make sure the path works after creating docker container.

File: neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py [hunk 5]
```diff
@@ -350,4 +365,25 @@ def _main(dfSub, dfRef, WMLSref, allz_num, allz, all_MuseROIs_name, spareAD, spa
...
+    dfSub = pd.read_pickle('/home/diwu/Desktop/2_dfSub.pkl')
+    WMLSref = pd.read_pickle('/home/diwu/Desktop/2_WMLSref.pkl')
+    allz_num = pd.read_pickle('/home/diwu/Desktop/2_allz_num.pkl')
+    allz = pd.read_pickle('/home/diwu/Desktop/2_allz.pkl')
+    all_MuseROIs_name = pd.read_pickle('/home/diwu/Desktop/2_all_MuseROIs_name.pkl')
+    spareAD = pd.read_pickle('/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/spare-calculation/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168_spareAD.pkl')
+    spareBA = pd.read_pickle('/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/spare-calculation/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168_spareBA.pkl')
+    pdf_path = '/home/diwu/Desktop/F1.pdf'
+    
+    _main(dfSub, dfRef, WMLSref, allz_num, allz, all_MuseROIs_name, spareAD, spareBA, pdf_path)
```
Comment (ashishsingh18) [inline neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py]: 1) Is this function needed after containerization?
2) Please remove all commented code.

File: neurodegeneration/processing-containers/roi_quantifier/files/roi_quantifier.py [hunk 6]
```diff
@@ -6,16 +6,18 @@
 from datetime import datetime
 import csv as _csv
 import os as _os
+  
+import SimpleITK as sitk
 
 #### Hardcoded reference data ###
 # Harmonized reference MUSE values #
-MUSE_Ref_Values = '/refs/combinedharmonized_out.csv'
+MUSE_Ref_Values = '../refs/combinedharmonized_out.csv'
```
Comment (ashishsingh18) [inline neurodegeneration/processing-containers/roi_quantifier/files/roi_quantifier.py]: Please make sure the path works from Docker container.

File: neurodegeneration/processing-containers/roi_quantifier/files/roi_quantifier.py [hunk 7]
```diff
@@ -6,16 +6,18 @@
...
 # Left and Right ROIs indices as well as ROI name to ROI number equivalency #
-maphemi = pd.read_csv('/refs/MUSE_ROI_Dictionary.csv')
+maphemi = pd.read_csv('../refs/MUSE_ROI_Dictionary.csv')
```
Comment (ashishsingh18) [inline neurodegeneration/processing-containers/roi_quantifier/files/roi_quantifier.py]: Please make sure the path works from Docker container.

File: neurodegeneration/processing-containers/roi_quantifier/files/roi_quantifier.py [hunk 8]
```diff
@@ -6,16 +6,18 @@
...
 # Single ROI to combined ROI mapping #
-MUSE_ROI_Mapping = '/refs/MUSE_DerivedROIs_Mappings.csv'
+MUSE_ROI_Mapping = '../refs/MUSE_DerivedROIs_Mappings.csv'
```
Comment (ashishsingh18) [inline neurodegeneration/processing-containers/roi_quantifier/files/roi_quantifier.py]: Please make sure the path works from Docker container.

File: neurodegeneration/processing-containers/roi_quantifier/files/roi_quantifier.py [hunk 9]
```diff
@@ -6,16 +6,18 @@
...
 # Harmonized reference WMLS values #
-WMLS_Ref_Values = '/refs/WMLS_combinedrefs.csv'
+WMLS_Ref_Values = '../refs/WMLS_combinedrefs.csv'
```
Comment (ashishsingh18) [inline neurodegeneration/processing-containers/roi_quantifier/files/roi_quantifier.py]: Please make sure the path works from Docker container.

File: neurodegeneration/processing-containers/roi_quantifier/files/roi_quantifier.py [hunk 10]
```diff
@@ -375,7 +377,7 @@ def _main(roi, icv, wmls, _json, out_path):
 	dfSub['Total White Matter Hyperintensity Volume'] = wmlsVol
 
 	### Add WMLS 604 reference datapoints to dfRef ###
-	WMLSref = pd.read_csv('/refs/WMLS_combinedrefs.csv').dropna()
+	WMLSref = pd.read_csv('../refs/WMLS_combinedrefs.csv').dropna()
```
Comment (ashishsingh18) [inline neurodegeneration/processing-containers/roi_quantifier/files/roi_quantifier.py]: Please make sure the path works from Docker container.

File: neurodegeneration/processing-containers/roi_quantifier/files/roi_quantifier.py [hunk 11]
```diff
@@ -413,4 +415,23 @@ def _main(roi, icv, wmls, _json, out_path):
...
+    # sitk.WriteImage(sitk.ReadImage(wmls), '/home/diwu/Desktop/kaapana-data-to-check-brainviz/kaapana_Seis/2.16.840.1.114362.1.12035716.24525521429.585200304.999.1428/wmls/2.16.840.1.114362.1.12035716.24525521429.585200304.999.1428.nii.gz')
+    
+    icv = ['/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/dlicv-inference/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168.nii.gz']
+    wmls = ['/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/wmls/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168.nii.gz']
+    
+    _json = '/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/GetT1Metadata/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168.json'
+    out_path = '/home/diwu/Desktop/2.pdf'
+    _main(roi, icv, wmls, _json, out_path)
+    
+
```
Comment (ashishsingh18) [inline neurodegeneration/processing-containers/roi_quantifier/files/roi_quantifier.py]: 1). Please remove all commented code.
2). Is this function needed after containerization?

**Review by ashishsingh18 — CHANGES_REQUESTED** [review]
@dwu12 Please see the comments. In general, please remove all commented code, make sure paths are correct after containerization.

File: neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py (hunk 3 above)
Comment (dwu12) [reply neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py]: done

File: neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py (hunk 4 above)
Comment (dwu12) [reply neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py]: done

File: neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py (hunk 5 above)
Comment (dwu12) [reply neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py]: done

**Review by ashishsingh18 — APPROVED** [review]
looks good.

**Pull request merged**